remote/new.go

```diff
@@ -60,6 +60,50 @@ func NewImage(repoName string, keychain authn.Keychain, ops ...imgutil.ImageOpti
 	}, nil
 }
 
+// NewIndex returns a new ImageIndex from the registry that can be modified and saved to local file system
+func NewIndex(repoName string, ops ...imgutil.IndexOption) (idx *ImageIndex, err error) {
```

Comment:
Similar comment to https://github.com/buildpacks/imgutil/pull/191/files#r1572576794, I don't think this function needs to know about a CNBIndex.

Reply:
Same comment as https://github.com/buildpacks/imgutil/pull/191#discussion_r1575392890